natives/toolchains.gradle

```diff
@@ -12,7 +13,26 @@ task buildWin32 {}
 task buildWin64 {}
 
 task buildDarwin {
-    dependsOn buildDarwin64
+    dependsOn buildDarwin64, buildDarwinAarch64
+
+    def newPath = new File(rootProject.projectDir, "src/main/resources/natives/" + "darwin" + "/")
+
+    def x86FilePath = new File(newPath, 'libtimescale-x86-64.dylib')
+    def aarch64FilePath = new File(newPath, 'libtimescale-aarch64.dylib')
+    def avx2FilePath = new File(newPath, 'libtimescale-avx2.dylib')
+    def finalFilePath = new File(newPath, 'libtimescale.dylib')
+
+    doLast {
+        def stdout = new ByteArrayOutputStream()
+        exec {
+            commandLine 'lipo', avx2FilePath.getAbsoluteFile(), aarch64FilePath.getAbsoluteFile(), '-output', finalFilePath.getAbsoluteFile(), '-create'
```

Comment:
Should be `x86FilePath` instead of `avx2FilePath`, since on systems without avx2 this file would be loaded as a fallback

Reply:
Thank you